stdlib/packages.go

```diff
@@ -19,6 +19,7 @@ import (
 	_ "github.com/influxdata/flux/stdlib/contrib/bonitoo-io/zenoss"
 	_ "github.com/influxdata/flux/stdlib/contrib/chobbs/discord"
 	_ "github.com/influxdata/flux/stdlib/contrib/jsternberg/influxdb"
+	- "github.com/influxdata/flux/stdlib/contrib/qxip/clickhouse"
```

Comment:
This will need to be an underscore (`_`) rather than a hyphen, to compile.

Reply:
Classic 3AM typo! Thanks for pointing it out @wolffcm